pyV2DL3/eventdisplay/IrfInterpolator.py

```diff
@@ -68,7 +69,12 @@ def __load_irf(self):
                 )
         self.irf_data = irf_data
         self.irf_axes = irf_axes
-        self.interpolator = RegularGridInterpolator(self.irf_axes, self.irf_data)
+        clk = click.get_current_context()
+        force_extrapolation = clk.params["force_extrapolation"]
+        if not force_extrapolation:
```

Comment:
If not ... else conditions are difficult to read, could you first to 'if force extrapolation and then the second part? The else loop is a 'double negation' at this point, always hard to parse.